package.json

```diff
@@ -77,16 +77,16 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^9.1.0",
+    "@angular/animations": "^11.2.5",
     "@angular/cdk": "^8.2.3",
-    "@angular/common": "^9.1.0",
-    "@angular/compiler": "^9.1.0",
-    "@angular/core": "^9.1.0",
+    "@angular/common": "^11.2.5",
+    "@angular/compiler": "^11.2.5",
+    "@angular/core": "^11.2.5",
     "@angular/fire": "5.2.1",
-    "@angular/forms": "^9.1.0",
-    "@angular/platform-browser": "^9.1.0",
-    "@angular/platform-browser-dynamic": "^9.1.0",
-    "@angular/router": "^9.1.0",
+    "@angular/forms": "^11.2.5",
+    "@angular/platform-browser": "^11.2.5",
+    "@angular/platform-browser-dynamic": "^11.2.5",
+    "@angular/router": "^11.2.5",
     "@fontsource/roboto": "4.2.1",
     "@ionic/angular": "^5.5.2",
     "@nrwl/angular": "^9.1.2",
```

Comment:
@nrwl/angular should probably also be bumped to `11.5.1`

Reply:
Agree 👍🏻